code/modules/cargo/goodies.dm

```diff
@@ -32,11 +32,11 @@
 	access_view = ACCESS_BRIG
 	contains = list(/obj/item/grenade/stingbang)
 
-/datum/supply_pack/goody/combatknives_single
-	name = "Combat Knife Single-Pack"
-	desc = "Contains one sharpened combat knive. Guaranteed to fit snugly inside any Nanotrasen-standard boot."
+/datum/supply_pack/goody/Survivalknives_single
+	name = "Survival Knife Single-Pack"
+	desc = "Contains one sharpened survival knive. Guaranteed to fit snugly inside any Nanotrasen-standard boot."
 	cost = PAYCHECK_HARD * 1.75
```

Comment:
I think you could just make this more expensive and keep it as is so that combat knives are still acquirable, just not freely by security.

Security being able to order armoury type stuff for free in general seems a bit strong anyways, it should probably just not be part of their free supply order

Reply:
> I think you could just make this more expensive and keep it as is so that combat knives are still acquirable, just not freely by security.
> 
> Security being able to order armoury type stuff for free in general seems a bit strong anyways, it should probably just not be part of their free supply order

This I think is a great option. The armory guns, knives, and other lethal items should require money, and the free ordering console could simply be for armor, batons, & such. If we went through with the proposed armor durability changes as well in future, I think it would balance out nicely.

Reply:
okay but sec already starts out with armor, batons, and such- they rarely need to buy more

Reply:
okay but I don't really care what sec start off with, they don't need free guns and mindshields too